src/node/historical_queries_interface.h

```diff
@@ -6,144 +6,100 @@
 #include "kv/store.h"
 #include "node/rpc/endpoint_registry.h"
 
+#include <chrono>
```

Comment:
Can this move to a newly created include/, which contains our public includes please?

Reply:
Done. This is now `include/ccf/historical_queries_interface.h`, and we have a global `include_directories(${CCF_DIR}/include)` in `common.cmake`.